### DISCUSSION TITLE
Receive data from external source

### DISCUSSION THREAD

**Discussion by guigsilva** [Budibase/budibase#4929] [Help]
Hello,

We are testing BB and what a great piece of software guys!!

We have a project where we will receive some data from POST in  xml, our partner will push this to us to an endpoint we will provide to them they alredy give us the xml schema.

The idea here is to create a table in BB with the xml structure plus some columns we need to work, receive the data from WS on BB and add to this table.

We try webhooks, and get some data in using jason, but seams we will not be able to receive the xml formated data...

We are now looking to the REST DS can this be done taking this way?

Thanks in advance

**Comment by mjashanks** [comment]
Hi There - welcome! Thanks for the kind words :)

Our APIs currently only support JSON. Our REST connector (outgoing) support XML, but I'm not sure if that's an option for you or not.

**Comment by guigsilva** [comment]
Hello again,

So to use XML we need to start the flow, right?

We willl try to understand with our partner if this is possible.

Thanks
Guilherme
